### DISCUSSION TITLE
Active Response - Manual Tigger scripts

### DISCUSSION THREAD

**Discussion by heinola** [Security-Onion-Solutions/securityonion#8440] [General] [answered]
In the prior SO version I have used I could issue a agent_control for lets say route-null to one or all my agents.  Is there a function like this still in so2 ?

**Comment by dougburks** [comment, marked as answer]
`agent_control` is still there, it's just inside the Wazuh container.  You should be able to execute it using `docker exec` similar to some of the examples at:
https://docs.securityonion.net/en/2.3/wazuh.html

**Comment by heinola** [comment]
Thank you all
